packages/lesmis/src/constants/constants.js

```diff
@@ -13,6 +13,7 @@ export const SINGLE_YEAR_GRANULARITY = 'one_year_at_a_time';
 export const MIN_DATA_DATE = '20150101';
 export const MIN_DATA_YEAR = '2015';
 export const DEFAULT_DATA_YEAR = '2020';
+export const DEFAULT_DATE_RANGE = { startDate: MIN_DATA_DATE, endDate: '20211231' };
```

Comment:
`20211231 ` is  a bit confusing to me, could you make that a named constant or add a comment to give a bit more background to the developer.

Reply:
Not needed